scripts/libmakepkg/buildenv/lto.sh.in

```diff
@@ -33,5 +33,10 @@ buildenv_lto() {
 	if check_option "lto" "y"; then
 		CFLAGS+=" -flto"
 		CXXFLAGS+=" -flto"
+		LDFLAGS+=" -flto"
```

Comment:
Why does it need to be added to LDFLAGS as well?

Reply:
according to [gcc-docs](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html)

```
-flto
...
To use the link-time optimizer, -flto and optimization options should be specified at compile time and during the final link.
```

Reply:
yeah, hm, just wondering why pacman doesn't do that by default..

Reply:
I guess because LDFLAGS is also passed to the compiler and not the linker, but it shouldn't hurt.

Reply:
https://gitlab.archlinux.org/pacman/pacman/-/commit/26ee6ff6adbcd8560ec3a65cf219d2ab98edb804

Reply:
ah, thanks :)